charts/seed-bootstrap-crds/charts/hvpa/templates/hvpa.yaml

```diff
@@ -243,6 +243,49 @@ spec:
                                   on a single metric (only `type` and one other matching
                                   field should be set at once).
                                 properties:
+                                  containerResource:
```

Comment:
Where are these changes for the HPA properties in the HVPA crds coming from? Did we pick up an HPA change by re-generating this crd?

Reply:
There are 2 Hvpa CRDs, [example/seed-crds/10-crd-autoscaling.k8s.io_hvpas.yaml](https://github.com/gardener/gardener/pull/5716/files/eed5f2d724acc672803725b84030b75ea1b47dee#diff-698b02bf865a5a3d0a68bd125bf2fca4429b22ded546c436a840587d029cdaca) and [charts/seed-bootstrap-crds/charts/hvpa/templates/hvpa.yaml](https://github.com/gardener/gardener/pull/5716/files/2a6f29a10d6732a72d1b4b56fb7ebb120b8e0b13#diff-e6c7533d6a6367b58400815a0791ef326bfa703171b4377850d40aec785e82b9). The first was autogenerated after my change to contain some small differences compared to its original state, I think this is a normal effect of increasing the version of the dependency. I fixed the other one manually to match the first one, after @rfranzke pointed out that there are some differences. Some of the differences (like the one you are pointing above) were not due to my changes, simply the second file was not well maintained and was a bit out of sync even before that.